### PR TITLE
chore: only run one spec with electron

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This should cut the time spent on electron considerably.

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -64,11 +64,14 @@ jobs:
...
       matrix:
-        browsers: [chrome, firefox, electron]
         node-version: [16.x]
         include:
           - browsers: electron
             spec: cypress/e2e/default/learn/challenges/projects.js
+          - browsers: chrome
+            spec: cypress/e2e/default/**/*.js
+          - browsers: firefox
+            spec: cypress/e2e/default/**/*.js
```
Comment (raisedadead) [inline .github/workflows/cypress.yml:74]: I might be wrong, but @Nirajn2311 had said something about the spec not getting picked up from actions. Correct?

**Comment by ojeytonwilliams** [comment]
I tried to be a bit too clever with this one and messed up the matrix syntax :face_exhaling:

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (ojeytonwilliams) [reply .github/workflows/cypress.yml:75]: He can confirm, but my understanding was that the list of tests gets filtered by the config and then again by the `--spec` parameter, if it exists.  We could try not passing it for chrome and firefox, but I wasn't sure how to do that.  @naomi-lgbt any ideas?

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

Screen Shot 2022-10-12 at 7 45 55 AM

The spec isn't very dynamic, but I'm fine with it for now. If we need more tests, it can be expanded.

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (naomi-lgbt) [reply .github/workflows/cypress.yml:75]:
```yml
if ${{ matrix.browsers}} === 'electron'
```

Maybe?

Comment (ojeytonwilliams) [reply .github/workflows/cypress.yml:75]: Do you want push a commit if it looks cleaner?

Comment (ojeytonwilliams) [reply .github/workflows/cypress.yml:75]: TM, indeed.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**